packages/contracts-core/script/configs/devnet/README.md

```diff
@@ -0,0 +1,13 @@
+# Configs
+
+[Messaging003AgentRoot.dc.json](Messaging003AgentRoot.dc.json) is automatically generated by the deploy script and contains proofs of the agent root representing the current state of each agent on each chain.
+[Messaging003.dc.json](Messaging003.dc.json) contains the configuration for the Messaging003 contract.
+    agents:
+        -  0: address of the guard(s).
+        -  other chains: address of the notary/ies for that chain
+    chainIdSummit: chainId of the Summit chain
```

Comment:
I am now realizing that naming this variable `chainIdSummit` was at least a little it  misleading. It was meant to describe the chain where `Summit` contract is deployed, and given that the test deployments were not featuring synapse chain, `synapseChainId` was not considered.

Perhaps it's time to update this?

Reply:
I think either is fine, but synapseChainId is much clearer in all testnet/mainnet environments.

Reply:
imo not as clear for devnets